src/Modules/OrchardCore.Commerce/Liquid/AddressFieldEditorViewModelConverterFilter.cs

```diff
@@ -0,0 +1,31 @@
+using Fluid;
```

Comment:
This filter is needed because otherwise it's not possible to override the Checkout.cshtml where the AddressFieldEditorViewModel is created in the view for the address fields. This way the filter gets the data to create the viewmodel from the CheckoutViewModel that's possible to get, and the other parameters are passed as arguments. 
E.g. `{{ Model | address_field_editor_view_model: address_field: orderPart.BillingAddress, city_name: "OrderPart.BillingAddress.Address.City" | shape_render }}`